app/models/card.rb

```diff
@@ -1,2 +1,13 @@
 class Card < ActiveRecord::Base
+    validates :original_text, presence: true
+    validates :translated_text, presence: true
+    validates :review_date, presence: true
+    validate :check_cards
+
+    def check_cards
+         if original_text.casecmp(translated_text) == 0
+            errors.add(:translated_text, "can't be the same as original text")    
```

Comment:
Line is too long. [82/80]<br>Use 2 (not 3) spaces for indentation.<br>Trailing whitespace detected.